src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleShell.cs

```diff
@@ -42,6 +43,11 @@ public static int Start(InitialSessionState initialSessionState, string bannerTe
                 throw PSTraceSource.NewArgumentNullException(nameof(args));
             }
 
+            if (helpText == null)
+            {
+                throw PSTraceSource.NewArgumentNullException(nameof(helpText));
```

Comment:
This is a breaking change.

Reply:
It is fixed bug. We don't expect a null for helpText
https://github.com/PowerShell/PowerShell/blob/63cf0c330c27c58b272ae0721359af725282d517/src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs#L641-L645

Reply:
After looking underlying code I think Write()/WriteLine() can accept null and we could make helpText nullable. Thoughts?

Reply:
Why do you think we don't? It's a public API, and could be used from outside PowerShell code.

Reply:
Made helpText nullable.
Done.